deploy/examples/bucket-topic.yaml

```diff
@@ -13,6 +13,7 @@ spec:
       uri: http://my-notification-endpoint:8080
     # uri: https://my-notification-endpoint:8443
       disableVerifySSL: true
+      cloudEvents: false # supported only in Ceph Quincy (v17) or newer
```

Comment:
Nit that this might be better named as `enableCloudEvents` or `enableCloudEventsSupport`.

Reply:
"cloudevents" are a sub-type of the HTTP endpoint - these are just HTTP messages with some extra header fields.
so maybe `sendCloudEvents` would be best?